diff --git a/node_modules/@orama/plugin-docusaurus-v3/dist/theme/SearchBar/index.js b/node_modules/@orama/plugin-docusaurus-v3/dist/theme/SearchBar/index.js
index eab29fc..2555aa0 100644
--- a/node_modules/@orama/plugin-docusaurus-v3/dist/theme/SearchBar/index.js
+++ b/node_modules/@orama/plugin-docusaurus-v3/dist/theme/SearchBar/index.js
@@ -36,6 +36,17 @@ export function OramaSearchNoDocs() {
                 where: formatSearchParams('current', collectionManager)
             } })));
 }
+export function getVersionName(version) {
+    if (!version) {
+        return undefined
+    }
+
+    if (typeof version === 'string') {
+        return version
+    }
+
+    return version.name
+}
 export function OramaSearchWithDocs({ pluginId }) {
     var _a;
     const colorMode = getColorMode();
@@ -44,7 +55,7 @@ export function OramaSearchWithDocs({ pluginId }) {
     const versions = useVersions(pluginId);
     const activeVersion = useActiveVersion(pluginId);
     const preferredVersion = getPreferredVersion(searchBoxConfig.basic.clientInstance);
-    const currentVersion = activeVersion || preferredVersion || versions[0];
+    const currentVersion = getVersionName(activeVersion) || getVersionName(preferredVersion) || getVersionName(versions[0]);
     const searchParams = {
         ...(currentVersion && {
             ...formatSearchParams(currentVersion, collectionManager)
